source/pool/contracts/Pool.sol

```diff
@@ -25,15 +25,15 @@ contract Pool is IPool, Ownable2Step {
   // Max percentage for a claim with infinite score
   uint256 public max;
 
-  // Mapping of tree root to boolean to enable claims
-  mapping(bytes32 => bool) public roots;
-
   // Mapping of address to boolean to enable admin accounts
   mapping(address => bool) public admins;
 
   // Mapping of tree root to account to mark as claimed
```

Comment:
Whilst we have now massively reduced the possibility of two roots that are the same, it could still happen (no change to voters or token holdings between votes).
The key here should be the `groupId`